Add example command to create /etc/apt/keyrings directory (#43626)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

   The same signing key is used for all repositories so you can disregard the version in the URL:

   ```shell

>    mkdir -p /etc/apt/keyrings

Reviewer: We should do one of:
- Move the note (line 208) here, and include a comment such as:
  ```suggestion
     # If you see an error, read the following note
  ```

- ```suggestion
     sudo mkdir -m 755 /etc/apt/keyrings
  ```
  (copied from line 208)
  For this option, let's also add a comment about _why_ this is needed (older Debian versions don't have that directory).

Author: updated

   The same signing key is used for all repositories so you can disregard the version in the URL:

   ```shell

>    # In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default; you can create it by the following mkdir command.
>    mkdir -p -m 755 /etc/apt/keyrings
>    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg

Reviewer 2: ```suggestion
   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
   # If you see an error then read the below note properly.
```

Author: @aj11anuj there are multiple notes in the section though.

   ```

> 

Reviewer 2: ```suggestion

{{< note >}}
In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings` and then re-run above curl cmd.
{{< /note >}}

```

   The same signing key is used for all repositories so you can disregard the version in the URL:

   ```shell

>    # If the folder `/etc/apt/keyrings` does not exist, it should be created before the curl command, read the note below.

Reviewer: ```suggestion
   # If the directory /etc/apt/keyrings does not exist, you should create it (see the note below).
```

   # sudo mkdir -p -m 755 /etc/apt/keyrings
   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
   ```
   
{{< note >}}

> In releases older than Debian 12 and Ubuntu 22.04, folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command.

Reviewer: ```suggestion
In releases older than Debian 12 and Ubuntu 22.04, the directory `/etc/apt/keyrings` does not exist by
default; if you don't have this directory, create it first.
```
